modules/database/variables.tf

```diff
@@ -57,10 +57,10 @@ variable "labels" {
 variable "database_version" {
   description = "Version for MySQL"
   type        = string
-  default     = "MYSQL_8_0_28"
+  default     = "MYSQL_8_0_30"
 
   validation {
-    condition     = contains(["MYSQL_5_7", "MYSQL_8_0", "MYSQL_8_0_28", "MYSQL_8_0_29"], var.database_version)
+    condition     = contains(["MYSQL_5_7", "MYSQL_8_0", "MYSQL_8_0_28", "MYSQL_8_0_30"], var.database_version)
```

Comment:
Sorry I had to short cut this PR. Buy maybe we could do regex her instead as a long term solution

https://developer.hashicorp.com/terraform/language/functions/regex

Reply:
`^MYSQL_(8_0(_[0-9]*)?|5_7)$`
You'll need to test it

Reply:
The inline below says this is valid. 

I inserted three invalid values to test if the regex would catch, and it worked.

```bash
for a in "MYSQL_5_7" "MYSQL_8_0" "MYSQL_8_0_28" "MYSQL_8_0_29" "MYSQL_8_0_30" "MYSQL_8_0_31" "MYSQL_8_1" "MYSQL_5_7_44" "MYSQL_8_0_30b" ; do echo "length(regexall(\"MYSQL_(8_0(_[0-9]*)?|5_7)$\", \"${a}\")) > 0" | terraform console; done
true
true
true
true
true
true
false
false
false
```

There is probably a better way to test, but this is valid.